### DISCUSSION TITLE
Map showing all the years QSOs

### DISCUSSION THREAD

**Discussion by magicbug** [magicbug/Cloudlog#735] [Ideas / Feature Requests]
**_Had this request via email_**

The option to have a map showing QSOs from the year as pins like the /logbook and dashboard maps.

**Comment by AndreasK79** [comment]
Perhaps this can be done in a way that you can select a bit what you want to show. I was thinking about something like the form I added for the KML export.

**Comment by magicbug** [comment]
Exactly what I was thinking, I like generating the maps for social media posts etc

**Comment by magicbug** [comment]
I've actually started this but its got some problems, one the way I'm loading the markers into Leaflet seems to have a long delay with 8000 QSOs then crashes the browser so need to figure a method out to load markers without that happening

The custom option /map/custom works OK but interface needs cleaned up

**Comment by magicbug** [comment]
Option might be to do it as a markerCluster (https://github.com/Leaflet/Leaflet.markercluster) but open to ideas.

**Comment by magicbug** [comment]
In fact this might be the simple solution https://github.com/eJuke/Leaflet.Canvas-Markers
